parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -1496,7 +1547,7 @@ loop = do
                   Nothing -> [] <$ respond (TermNotFound' . SH.take hqLength . Reference.toShortHash $ Reference.DerivedId rid)
                   Just tm -> do
                     respond $ TestIncrementalOutputStart ppe (n,total) r tm
-                    tm' <- eval $ Evaluate1 ppe tm
+                    tm' <- eval $ Evaluate1 ppe False tm
```

Comment:
Should this (and the one on line 1627) be `True`? Or is `False` correct?

Reply:
I wish I could see more context for this. I see like 2 surrounding lines with no way to expand that on mobile...

Reply:
This one is fine - the test watch is added to the cache explicitly later. I’ll check the other one when by a computer.

Reply:
Yeah the other is okay too, it's an I/O test which you don't want to cache. I'm adding a comment in the code to this effect.